tests/integration/network.go

```diff
@@ -86,6 +88,10 @@ func (*Network) SynchronizeNetwork(ctx context.Context) error {
 	return nil
 }
 
+func (n *Network) FundedAuthFactory() chain.AuthFactory {
+	return n.Configuration().AuthFactories()[0]
+}
```

Comment:
Seems like it would make more sense to relay the exact return type rather than only support returning the value at index 0? Alternatively, we could just have tests access the auth factory directly through the config rather than adding a helper function to the test network here

Reply:
I removed the helper function 👍